pkg/services/ngalert/notifier/channels/slack.go

```diff
@@ -52,17 +52,22 @@ func NewSlackNotifier(model *NotificationChannelConfig, t *template.Template, fn
 		return nil, receiverInitError{Cfg: *model, Reason: "no secure settings supplied"}
 	}
 
+	endpointURL := model.Settings.Get("endpointUrl").MustString()
+	if endpointURL == "" {
+		endpointURL = SlackAPIEndpoint
+	}
```

Comment:
```suggestion
	endpointURL := model.Settings.Get("endpointUrl").MustString(SlackAPIEndpoint)
```
`MustString` takes a variadic arg, which is the default to use if the key does not exist.